app/views/testing_grounds/show.html.haml

```diff
@@ -1,3 +1,9 @@
+- if display_invalid_message?
```

Comment:
I think it would be better to validate the LES prior to performing a calculation (requesting `data.json`), instead of every time we open testing_grounds/show.

data.json could return one of:

* The load data, if available.
* `{ pending: true }`
* `{ error: 'message' }`